packages/client/hmi-client/src/App.vue

```diff
@@ -118,4 +121,17 @@ main {
 .data-explorer {
 	z-index: 3;
 }
+
+footer {
+	width: 100%;
+	height: 48px;
+	background-color: #344054;
```

Comment:
Could you use a CSS property for it? One of the `--grey-X0`

Reply:
we don't have an existing property but I will create one

Reply:
or do you just want to use an existing colour? I grabbed the colour directly from figma

Reply:
`--gray-800` should be the closest one.